### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by phelian23**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="styles.css" rel="stylesheet"><title>Hello World</title>
+    </head>
+    <body>
+        <h1 class="header">Hello Microverse!</h1>
+    </body>
+</html>
```
Comment (dasileker) [inline index.html]: - [ ] This is causing linters error, please add  linters to your project to fix the issue before send a PR for review [ HTML/CSS linters link](https://github.com/microverseinc/linters-config)

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (dasileker) [inline test.md]: - [ ] the test.md should not be visible after adding it to the gitignore file.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.header {
+    color: blue;
```
Comment (dasileker) [inline styles.css]: - [ ] Also your style is having linters error so doo the same here, add style linter file to fix all issue before sending it for review.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @phelian23,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — APPROVED** [review]
### Hi @phelian23, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 

OPTIONAL

- [ ] Add a PR description tp your PR explaining all the features you have implemented.
- [ ] Customize your readme file to suit your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**